app/components/application_complete_content_component.html.erb

```diff
@@ -1,17 +1,12 @@
-<p class="govuk-body">
-  You submitted your application on <%= submitted_at_date %>. Training providers must
-  respond to you by <%= respond_by_date %>.
-</p>
-
 <% if @dates.form_open_to_editing? %>
   <div class="govuk-inset-text">
     <p class="govuk-body">
-      You have <%= formatted_days_remaining %> (until <%= edit_by_date %>) to withdraw your application and
-      resubmit it to a different course or provider. You’ll be able to edit
-      your application before you resubmit.
+      You have <%= render(ApplicationDateComponent, application_form: @application_form, type: :days_remaining).squish %>
```

Comment:
Is `squish` necessary? maybe can we move it inside the component,
it is a bit noisy to have `.squish` after renders.

Reply:
Yes; the component returns a buffer which is `'15 May 2019\n'`. Removing the `\n` is not necessary in this exact case, but is necessary in cases like:

```
Application submitted on 18 November 2019 .
                                         ^~~~~ annoying space!
```